cfme/tests/automate/test_service_automate.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.base.credential import Credential
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.blockers import BZ
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log_validator import LogValidator
+
+
+pytestmark = [test_requirements.automate]
+
+
+@pytest.fixture(scope="module")
+def new_users(appliance):
+    """This fixture creates new users"""
+    users = []
+    for i in range(2):
+        user = appliance.collections.users.create(
+            name="user_{}".format(fauxfactory.gen_alphanumeric().lower()),
+            credential=Credential(principal='uid{}'.format(fauxfactory.gen_alphanumeric(4)),
+                                  secret=fauxfactory.gen_alphanumeric(4)),
+            email=fauxfactory.gen_email(),
+            groups=appliance.collections.groups.instantiate(
+                description="EvmGroup-super_administrator"),
+            cost_center="Workload",
+            value_assign="Database",
+        )
+        users.append(user)
+    yield users
+    if not BZ(1720273).blocks and user in users:
```

Comment:
Shouldn't this BZ block the test from running?

Reply:
- No. This BZ doesn't have any relationship with test case. 
- It gives problem while doing teardown of custom users in 5.11.

Reply:
Right, if the tear down for the test is unable to occur, shouldn't we block the test case? Or is there no harm in having more than one user on an appliance after this test completes?

Reply:
I am not sure about the blocking the test case only because of teardown.
According to me, there will not be any harm if more than one user in the appliance.

Reply:
Discussed offline about cleaning up via a REST query, that seems like the best option while this BZ is blocking.